src/CommunityToolkit/Xamarin.CommunityToolkit/Behaviors/Animations/AnimationTypes/BounceAnimation.shared.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Threading.Tasks;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Behaviors
+{
+	public class BounceInAnimation : AnimationBase
+	{
+		protected override uint DefaultDuration { get; set; } = 300;
+
+		public override async Task Animate(View view) =>
+			await Task.Run(() =>
+			{
+				Device.BeginInvokeOnMainThread(() =>
+				{
+					view.Animate("BounceIn", BounceIn(view), 16, Duration);
+				});
+			});
+
+		internal Animation BounceIn(View view)
+		{
+			var animation = new Animation();
+
+			animation.WithConcurrent(
+				f => view.Scale = f,
+				0.5, 1,
+				Easing.Linear, 0, 1);
+
+			animation.WithConcurrent(
+				(f) => view.Opacity = f,
+				0, 1,
+				null,
+				0, 0.25);
+
+			return animation;
+		}
+	}
+
+	public class BounceOutAnimation : AnimationBase
+	{
+		protected override uint DefaultDuration { get; set; } = 300;
+
+		public override async Task Animate(View view) =>
+			await Task.Run(() =>
```

Comment:
```suggestion
		public override Task Animate(View view) =>
			Task.Run(() =>
```

Reply:
Also, I believe there's a property on Task.Run that makes it run on UI Thread, that way we don't need to call the `BeginInvokeOnMainThread´